error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/StreamRules.java

```diff
@@ -624,4 +625,28 @@ R after(
       return stream.flatMap(mapper).collect(collector);
     }
   }
+
+  static final class StreamConcat<T> {
+    @BeforeTemplate
+    Stream<T> before(Stream<T> s1, Stream<T> s2) {
+      return Stream.of(s1, s2).flatMap(Function.identity());
+    }
+
+    @AfterTemplate
+    Stream<T> after(Stream<T> s1, Stream<T> s2) {
+      return Stream.concat(s1, s2);
+    }
+  }
```

Comment:
This might be redundant if `StreamsConcat` turns `Stream.of(s1, s2).flatMap(Function.identity())` into `Streams.concat(s1, s2)` and `ConcatTwoStreams` turns it into `Stream.concat(s1, s2)`

Reply:
Dropped the rule 👍